config/root.rc.in

```diff
@@ -0,0 +1,48 @@
+#include <winver.h>
+
+ROOT_ICON   ICON   "../icons/RootIcon.ico"
+
+#define VER_FILEVERSION             @ROOT_MAJOR_VERSION@,@ROOT_MINOR_VERSION@,@ROOT_PATCH_VERSION@,0
+#define VER_FILEVERSION_STR         "@ROOT_MAJOR_VERSION@.@ROOT_MINOR_VERSION@.@ROOT_PATCH_VERSION@.0\0"
+
+#define VER_PRODUCTVERSION          @ROOT_MAJOR_VERSION@,@ROOT_MINOR_VERSION@,@ROOT_PATCH_VERSION@,0
+#define VER_PRODUCTVERSION_STR      "@ROOT_MAJOR_VERSION@.@ROOT_MINOR_VERSION@.@ROOT_PATCH_VERSION@.0"
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Version
+//
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION @ROOT_MAJOR_VERSION@,@ROOT_MINOR_VERSION@,@ROOT_PATCH_VERSION@,0
+ PRODUCTVERSION @ROOT_MAJOR_VERSION@,@ROOT_MINOR_VERSION@,@ROOT_PATCH_VERSION@,0
+ FILEFLAGSMASK 0x3fL
+#ifdef _DEBUG
+ FILEFLAGS 0x1L
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS 0x10004L
+ FILETYPE 0x1L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "Comments", "\0"
+            VALUE "CompanyName", "CERN\0"
+            VALUE "FileDescription", "ROOT executable\0"
+            VALUE "FileVersion", "@ROOT_MAJOR_VERSION@.@ROOT_MINOR_VERSION@.@ROOT_PATCH_VERSION@.0\0"
+            VALUE "InternalName", "\0"
+            VALUE "LegalCopyright", "Copyright (c) 2020\0"
```

Comment:
Can we parametrize the current year? We will totally forget to update that in 2021...

Reply:
I'll see what I can do

Reply:
OK, found a solution